package.json

```diff
@@ -31,5 +35,11 @@
     "@vue/compiler-sfc": "^3.0.0",
     "@vue/server-renderer": "^3.0.0",
     "merge-options": "^3.0.3"
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.7",
+    "vite": "^1.0.0-rc.9",
+    "vue": "^3.0.2",
+    "vue-router": "^4.0.0-rc.3"
   }
```

Comment:
had to install all those for types, since i couldn't get the right @types files...

Reply:
I remember I tried installing them as devDependencies and then things blew up. Hope it's not the case this time 😅

Reply:
it seemed to work fine for me when testing the branch. Do you remember exactly what blew up? Maybe you can test this branch locally too?

Reply:
@tbgse Just tested and I couldn't make it work. The commonjs build seems wrong for some reason. It throws `Cannot read property 'bind' of undefined`and other errors after that one is fixed.
Can you try to build vite-ssr and run `yarn refresh` and `yarn build` in `example` folder? 🤔 

Apart from that, it's missing TypeScript devDependency and the `package.json` should also be copied to `dist` during development (otherwise the binary is not copied, so `vite-ssr build` command fails).